### PR TITLE
Fix API usage in corpus tests

### PR DESCRIPTION
**Pull request by mbj**
* I HATE DYNLANGS!

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb Ready for review.

File: spec/support/corpus.rb [hunk 1]
```diff
@@ -131,7 +131,7 @@ def install_mutant
         end
         lockfile = repo_path.join('Gemfile.lock')
         lockfile.delete if lockfile.exist?
-        system('bundle install')
+        system(%w[bundle install])
```
Comment (dkubb) [inline spec/support/corpus.rb]: The `install` comment is optional. You could just say `%w[bundle]` if you wanted.

Comment (mbj) [inline spec/support/corpus.rb]: Good catch.

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Pull request merged**